quadriga/src/main/java/edu/asu/diging/quadriga/core/model/EventGraph.java

```diff
@@ -1,19 +1,29 @@
 package edu.asu.diging.quadriga.core.model;
 
+import java.time.LocalDateTime;
+
 import org.bson.types.ObjectId;
 
+import edu.asu.diging.quadriga.api.v1.model.Context;
 import edu.asu.diging.quadriga.core.model.events.CreationEvent;
 
 public class EventGraph {
 
     private ObjectId _id;
     private CreationEvent  rootEvent;
     private DefaultMapping defaultMapping;
+    private LocalDateTime creationTime;
+    private ObjectId collectionId;
+    private String appName;
```

Comment:
what's this for?

Reply:
This is as per the ticket description - 
"info about last submitted network to that collection (when, from whom)"
For the "From whom" part, we need to display appName

**On slack we had this discussion:**

Julia Lessios-Damerow  4:08 PM
We probably need to add a story to get info about just one app. But the story Maulik works on would give you a list of all apps that you could also use. But I create a ticket for getting info about one app.
4:08
And sounds good, have a good evening.:+1:
Pooja Kulkarni  4:09 PM
Oh okay okay
So I’ll add a comment that once that ticket is done, app name should be displayed on collections page
Thanks and have a nice day!

Julia Lessios-Damerow  4:10 PM
:+1:


The "comment" I am referring to in the above discussion is the one I have added in the AddNetworkApiController:
`
            /**
             * This is a temporary solution as per the comment on story Q20-18
             * A new story will later be created to get info about just one app from citesphere using OAuth token.
             * This app's name should be stored in eventGraph instead of the client id
             */
`

Reply:
but this would refer to the app, right? so we need the id of the app or something? also, the variable name should indicate what is referred to here. Maybe "submittingApp" or something.